app/components/PageHead.tsx

```diff
@@ -31,18 +35,16 @@ export const PageHead = (props: PageHeadProps) => {
       <meta name="description" content={props.metaDescription} />
       <meta property="og:description" content={props.metaDescription} />
       <meta property="og:title" content={props.mediaTitle} />
-      {props.mediaImageSrc && (
-        <meta property="og:image" content={props.mediaImageSrc} />
-      )}
+      <meta property="og:image" content={mediaImageSrc} />
+
```

Comment:
do you think it causes problems if the page has no `mediaImageSrc`? like rendering an empty meta og:image tag, is that handled gracefully by most social media sites?

Reply:
Hi! 

I coded the getOgImageSrc function so that it returns `urls.mediaImage` if nothing is provided.
I though it was a nice addition, but maybe not. Let me know.